frontera/tests/run_zmq_broker.sh

```diff
@@ -1,2 +1,18 @@
 #!/usr/bin/env bash
-python -m frontera.contrib.messagebus.zeromq.broker 2> broker.log &
\ No newline at end of file
+# Start ZMQ broker with hostname instead of IP
+python -m frontera.contrib.messagebus.zeromq.broker --address localhost 2>> broker.log &
+sleep 5
```

Comment:
Wait, wait, wait :)
This is not reliable. Travis VMs are suffer from constantly changing performance. So, you never know for how long broker needs to be running.
Let's better run several broker instances using different ports till the end of test?